src/psioptionseditor.cpp

```diff
@@ -135,11 +135,18 @@ PsiOptionsEditor::PsiOptionsEditor(QWidget *parent) : QWidget(parent)
     tpm_ = new QSortFilterProxyModel(this);
     tpm_->setSourceModel(tm_);
 
-    QVBoxLayout *layout = new QVBoxLayout(this);
-    layout->setContentsMargins(0, 0, 0, 0);
+    QVBoxLayout *mainLayout = new QVBoxLayout(this);
+    mainLayout->setContentsMargins(10, 10, 10, 10); // Adjust margins for the main layout
```

Comment:
I believe for the detached window you can avoid setting margins at all. 
I guess defaults should be good enough.